bigtable/client/idempotent_mutation_policy_test.cc

```diff
@@ -21,7 +21,7 @@ TEST(IdempotentMutationPolicyTest, Simple) {
   using namespace bigtable::chrono_literals;
   auto policy = bigtable::DefaultIdempotentMutationPolicy();
   EXPECT_TRUE(policy->is_idempotent(
-      bigtable::DeleteFromColumn("fam", "col", 0, 10).op));
+      bigtable::DeleteFromColumn("fam", "col", 0_ms, 10_ms).op));
```

Comment:
Why did you change the values from microseconds to milliseconds in these tests?  Just curious, it does not matter but I wonder if something interesting was going one.

Reply:
I thought number we passing, is default milliseconds, so I added _ms.
So if default we pass as micro seconds, then we can use  10_us instead of 10_ms, am I correct ?

Reply:
Yup, the previous values (as raw integers) represented microseconds. Using `10_us` would require less changes in the tests.

Reply:
OK, then I will change every where _us,  where I have used _ms for raw integers

Reply:
You can use the units that are most natural for each test.  For example, if the test used to say `2000`, that can now be represented as `2_ms` or `2000_us`.  The former is probably more readable.  That is the whole point of supporting `std::duration<>`: to make the code easy to understand.